### PR TITLE
Update debugfs plugin metadata.

### PR DESCRIPTION
**Pull request by Ferroin**
##### Summary

Updated by-hand based on individual knowledge of the areas in question.

This includes adding some info to a handful of fields we’re not actually rendering in all cases yet to save time later when we go through this properly.

### CODE REVIEW

File: collectors/debugfs.plugin/multi_metadata.yaml [hunk 1]
```diff
@@ -100,7 +112,7 @@ modules:
                 - name: order9
                 - name: order10
             - name: mem.fragmentation_index_normal
-              description: Memory fragmentation index for each order
+              description: Memory fragmentation index for each order within the Normal zone
```
Comment (ilyam8) [inline collectors/debugfs.plugin/multi_metadata.yaml:115]: `description` reflects the chart title

Comment (Ferroin) [reply collectors/debugfs.plugin/multi_metadata.yaml:115]: The chart titles need updated then, they’re not properly descriptive of what each chart is.

Comment (Ferroin) [reply collectors/debugfs.plugin/multi_metadata.yaml:115]: I can add that as a separate PR, but we should be flagging the memory zone in the chart title/description.

**Comment by Ferroin** [comment]
Rebased to resolve merge conflicts.

**Review by Ancairon — APPROVED** [review]

**Pull request merged**